service/create/ports.go

```diff
@@ -26,5 +26,7 @@ func extractWorkersSecurityGroupPorts(cluster awstpr.CustomObject) []int {
 		cluster.Spec.Cluster.Kubernetes.Kubelet.Port,
 		sshPort,
 		calicoBGPNetworkPort,
+		httpsPort, // TODO Move the https and http ports to a separate security group for Ingress ELB.
```

Comment:
When you go like that the whole list has to be reordered. Lets not go along this road.